setup.py

```diff
@@ -109,9 +109,9 @@ def run(self):
       classifiers=classifiers,
       cmdclass={'build_py': CustomPythonBuild},
       install_requires = ['typed-ast >= 1.1.0, < 1.2.0',
-                          'psutil >= 5.4.0, < 5.5.0',
                           ],
       extras_require = {
           ':python_version < "3.5"': 'typing >= 3.5.3',
+          'dmypy': 'psutil >= 5.4.0, < 5.5.0',
```

Comment:
You should add `:sys_platform!="win32"` after `dmypy`, since it is not used on Windows.